### ISSUE TITLE
您好！请问一下这个对模型的改进，可以用于目标检测吗？

### ISSUE THREAD

**Issue by Ajoker0524** [mkang315/ASF-YOLO#7]

**Comment by mkang315** [comment]
Of course. ASF-YOLO can apply to both object detection and instance segmentation.

**Comment by Ajoker0524** [comment]
ok， thank you very much! If I use this code, I would still directly refer to the readme.md document for training and prediction?
